website/static/js/koHelpers.js

```diff
@@ -69,6 +69,28 @@ var sanitizedObservable = function(value) {
     });
 };
 
+// maps js data one deep to observables
```

Comment:
Might want to document the `exclude` for clarification. It doesn't doesn't exclude the items from the output, it just excludes them from being turned into observables.